src/rulesets/oas3/__tests__/valid-example.ts

```diff
@@ -215,3 +216,95 @@ describe('valid-example', () => {
     },
   );
 });
+
+describe('valid-schema-example', () => {
```

Comment:
Looks like we've split this one rule into two but have left them both in the old file? The file names match the rules so we should have:

- src/rulesets/oas3/__tests__/valid-openapi-example.ts
- src/rulesets/oas3/__tests__/valid-schema-example.ts

Reply:
If not in `src/rulesets/oas3/ruleset.json`, then where I should put these 2 new rules? Thought `oas3/ruleset.json` is the place as the two rules belong to oas3 validators, no?

Reply:
The valid-example.ts is when there was one rule. You are splitting it into two rules. So make two files, the ones above, and tadaaaa you’re done. No new ruleset, that’s a total misuse of ruleset a.

1 rule becomes 2 rules.